config/initializers/audited_hacks.rb

```diff
@@ -19,9 +19,40 @@ module AuditHacks
     alias_attribute :association_id, :associated_id
     alias_attribute :association_type, :associated_type
 
+    after_commit :log_to_stdout, on: :create, if: :logging_to_stdout?
+
     def self.delete_old
       where('created_at < ?', TTL.ago).delete_all
     end
+
+    def self.logging_to_stdout?
+      Features::LoggingConfig.config.audits_to_stdout
+    end
+
+    delegate :logging_to_stdout?, to: :class
+
+    def log_to_stdout
+      Rails.logger.info log_trail
```

Comment:
I think you should use Tagged Logger

https://api.rubyonrails.org/v4.2.11.1/classes/ActiveSupport/TaggedLogging.html